tasks/timemachine.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Time Machine Directories
+  file:
+    path: "{{ timemachine_data_directory }}"
+    state: directory
+
+- name: Time Machine Docker Container
+  docker_container:
+    name: timemachine
+    image: mbentley/timemachine
+    pull: true
+    ports:
+      - "548:548"
+      - "5353:5353"
```

Comment:
What is this port for?

Reply:
5353 is for mDNS.  I retested and it doesn't seem to be required.  Removed.